Netcracker/KubeMarine
# [PSUPCLPL-13377] - Guide For OAuth2 authorization in Kubernetes

### Description
* We use kube-bench to make KubeMarine CIS aligned. Some of the improvements do not support in source code and could be applied manually.
* The suggested improvements have be reproduced
* Some additional observation in CIS scan which needs to be added in the hardening guide 

### Solution
* Update the guide (hardening) in KubeMarine documentations


### How to apply
Not applicable


### Test Plan
After applying the changes mentioned in this PR, run the kube-bench scan and verify result for identifier **3.1.2**

- Login to one of the master node of the cluster on which it is going to be tested
- Download `kube-bench_0.6.17_linux_amd64` package from `https://github.com/aquasecurity/kube-bench`
- Install downloaded pkg using `sudo dpkg -i kube-bench_0.6.17_linux_amd64.deb`
- Run scan using command - `kube-bench` and check for identifier **3.1.2**


### Checklist
